modules/Movecraft/src/main/resources/types/Airship.craft

```diff
@@ -0,0 +1,141 @@
+name: Airship
+maxSize: 5000
+minSize: 1000
+allowedBlocks: 
+    - "#planks"
+    - water
+    - lava
+    - "#logs"
+    - "#glass"
+    - lapis_block
+    - dispenser
+    - note_block
+    - "#beds"
+    - sticky_piston
+    - piston
+    - piston_head
+    - "#wool"
+    - gold_block
+    - iron_block
+    - "#slabs"
+    - bricks
+    - bookshelf
+    - torch
+    - fire
+    - "#stairs"
+    - "#chests"
+    - redstone_wire
+    - diamond_block
+    - crafting_table
+    - furnace
+    - blast_furnace
+    - smoker
+    - "#signs"
+    - "#doors"
+    - ladder
+    - lever
+    - "#pressure_plates"
+    - redstone_torch
+    - redstone_wall_torch
+    - "#buttons"
+    - clay
+    - "#fences"
+    - netherrack
+    - glowstone
+    - cake
+    - repeater
+    - "#trapdoors"
+    - "#stone_bricks"
+    - iron_bars
+    - "#fence_gates"
+    - nether_brick
+    - enchanting_table
+    - brewing_stand
+    - cauldron
+    - end_stone
+    #- dagon_egg does not work for some reason???
+    - redstone_lamp
+    - ender_chest
+    - tripwire_hook
+    - tripwire
```

Comment:
this is actually a change in behaviour but i don't mind it too much

Reply:
I figured having tripwire hooks was useless without the tripwires, I should have mentioned it in the PR description